project/OSGi.scala

```diff
@@ -44,6 +44,43 @@ object OSGi {
 
   val remote = exports(Seq("akka.remote.*"))
 
+  val parsing = exports(Seq("akka.parboiled2.*", "akka.shapeless.*"),
+    imports = Seq(optionalResolution("scala.quasiquotes")))
+
+  val httpCore = exports(Seq("akka.http.*"),
+    imports = Seq(streamAndHttpImport("akka.stream.*"),
+      streamAndHttpImport("akka.parboiled2.*"),
+      streamAndHttpImport("akka.shapeless.*")))
+
+  val http = exports(Seq("akka.http.impl.server",
+    "akka.http.scaladsl.server.*", "akka.http.javadsl.server.*",
+    "akka.http.scaladsl.client", "akka.http.scaladsl.coding", "akka.http.scaladsl.common",
+    "akka.http.scaladsl.marshalling", "akka.http.scaladsl.unmarshalling"),
+    imports = Seq(streamAndHttpImport("akka.stream.*"),
+      streamAndHttpImport("akka.http.*"),
+      streamAndHttpImport("akka.parboiled2.*")))
+
+  val httpTestkit = exports(Seq("akka.http.scaladsl.testkit.*", "akka.http.javadsl.testkit.*"),
+    imports = Seq(streamAndHttpImport("akka.stream.*"),
+      streamAndHttpImport("akka.http.*")))
+
+  val httpSprayJson = exports(Seq("akka.http.scaladsl.marshallers.sprayjson"),
+    imports = Seq(streamAndHttpImport("akka.stream.*"),
+      streamAndHttpImport("akka.http.*")))
+
+  val httpXml = exports(Seq("akka.http.scaladsl.marshallers.xml"),
+    imports = Seq(streamAndHttpImport("akka.stream.*"),
+      streamAndHttpImport("akka.http.*")))
+
+  val httpJackson = exports(Seq("akka.http.javadsl.marshallers.jackson"),
+    imports = Seq(streamAndHttpImport("akka.stream.*"),
+      streamAndHttpImport("akka.http.*")))
+
+  val stream = exports(Seq("akka.stream.*"))
+
+  val streamTestkit = exports(Seq("akka.stream.testkit.*"),
+    imports = Seq(streamAndHttpImport("akka.stream.*")))
```

Comment:
As far as I understand it the plugin understands project dependencies.

For reference here's a MANIFEST from current state of PR (removed those explicit imports) for testkit:

```
[1] <y2/local/com.typesafe.akka/akka-stream-testkit_2.11/2.4-SNAPSHOT/jars/akka-stream-testkit_2.11.jar [-][RO][tar]                                                                                                                                                                                        5,1-1           1%
  1 Manifest-Version: 1.0
  2 Bnd-LastModified: 1452646866873
  3 Bundle-ManifestVersion: 2
  4 Bundle-Name: com.typesafe.akka.stream.testkit
  5 Bundle-SymbolicName: com.typesafe.akka.stream.testkit
  6 Bundle-Version: 2.4.0.SNAPSHOT
  7 Created-By: 1.8.0_65 (Oracle Corporation)
  8 Export-Package: akka.stream.testkit;uses:="akka.actor,akka.stream,akka.s
  9  tream.impl,akka.testkit,org.reactivestreams,scala,scala.collection,scal
 10  a.collection.immutable,scala.concurrent.duration,scala.reflect,scala.ru
 11  ntime,scala.util";version="2.4.0.SNAPSHOT",akka.stream.testkit.javadsl;
 12  uses:="akka.actor,akka.stream.javadsl,akka.stream.testkit,scala.reflect
 13  ";version="2.4.0.SNAPSHOT",akka.stream.testkit.scaladsl;uses:="akka.act
 14  or,akka.stream.scaladsl,akka.stream.testkit,scala.reflect";version="2.4
 15  .0.SNAPSHOT"
 16 Import-Package: akka.actor;version="[2.4,2.5)",akka.stream;version="[2.4
 17  ,2.5)",akka.stream.impl;version="[2.4,2.5)",akka.stream.javadsl;version
 18  ="[2.4,2.5)",akka.stream.scaladsl;version="[2.4,2.5)",akka.testkit;vers
 19  ion="[2.4,2.5)",scala;version="[2.11,2.12)",scala.collection;version="[
 20  2.11,2.12)",scala.collection.generic;version="[2.11,2.12)",scala.collec
 21  tion.immutable;version="[2.11,2.12)",scala.collection.mutable;version="
 22  [2.11,2.12)",scala.concurrent.duration;version="[2.11,2.12)",scala.refl
 23  ect;version="[2.11,2.12)",scala.runtime;version="[2.11,2.12)",scala.uti
 24  l;version="[2.11,2.12)",org.reactivestreams;version="[1.0,2)"
 25 Include-Resource: /Users/ktoso/code/akka/akka-stream-testkit/src/main/re
 26  sources,/Users/ktoso/code/akka/akka-stream-testkit/target/resource_mana
 27  ged/main
 28 Private-Package: com.typesafe.akka.stream.testkit.*
 29 Tool: Bnd-2.1.0.20130426-122213
```

Calling it a night though now, will analyse in the morning